app/components/SocialsList.tsx

```diff
@@ -5,6 +5,7 @@ import { DiscordLogo } from "./logos/DiscordLogo";
 import { LinkedInLogo } from "./logos/LinkedInLogo";
 import { MastodonLogo } from "./logos/MastodonLogo";
 import { TwitterLogo } from "./logos/TwitterLogo";
+import { GitHubLogo } from "./logos/GitHubLogo";
```

Comment:
You need to sort these imports as such:

```suggestion
import { GitHubLogo } from "./logos/GitHubLogo";
import { TwitterLogo } from "./logos/TwitterLogo";
```